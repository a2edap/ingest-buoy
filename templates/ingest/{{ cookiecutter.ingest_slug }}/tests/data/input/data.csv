# TODO â€“ Developer: Replace this file with your actual sample data. Please try to keep
# this file small, as it will be committed to the repository and used in regular tests.
Timestamp (end of interval),Example
"2021-11-14 22:52:00",0.5
"2021-11-14 22:55:00",0.6
"2021-11-14 22:57:00",0.7
"2021-11-14 22:59:00",0.8
"2021-11-14 23:01:00",0.5
"2021-11-14 23:03:00",1.0