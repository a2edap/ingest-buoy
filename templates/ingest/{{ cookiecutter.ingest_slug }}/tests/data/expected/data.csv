# TODO â€“ Developer: Replace this file with your expected output data. This will be used
# to verify that your ingest is working as intended as other changes are made to this
# repository.